pkg/readers/lockfile_reader.go

```diff
@@ -26,11 +26,16 @@ func (p *lockfileReader) Name() string {
 	return "Lockfiles Based Package Manifest Reader"
 }
 
+func (p *lockfileReader) ApplicationName() (string, error) {
+	return "vet-scanned-project", nil
```

Comment:
Can we have a `const` defined for a default app name that can be used by all readers that need it? May be just have a `common.go` in `readers` package for common stuff

Reply:
yep, thats better